hadoop-crypto/src/main/java/com/palantir/hadoop/EncryptedFileSystem.java

```diff
@@ -76,6 +79,29 @@ public FSDataOutputStream create(Path path, FsPermission permission,
         return os;
     }
 
+    @Override
+    public boolean rename(Path src, Path dst) throws IOException {
```

Comment:
`Path.toString()` returns the canonical path string so `new Path("bar//baz").toString()` gives `"bar/baz"`. that's what we use to put/get keys so that should be fine. anything else is on the underlying filesystem